### ISSUE TITLE
sample networks with higher input resolutions

### ISSUE THREAD

**Issue by Mshz2** [VascoLopes/EPE-NAS#2]
Hi there,
I would appreciate if you could guide me how can sample netwrok configurations from NAS-Bench while increasing the input size of networks.
For instance all sampled netwrok configurations by `network = get_cell_based_tiny_net(config)` are having input size of 16 for con2d in their first few blocks:

> TinyNetwork(
  TinyNetwork(C=16, N=5, L=17)
  (stem): Sequential(
    (0): Conv2d(3, **_16_**, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)
    (1): BatchNorm2d(16, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
  )
  (cells): ModuleList(
    (0): InferCell(
      info :: nodes=4, inC=16, outC=16, [1<-(I0-L0) | 2<-(I0-L1,I1-L2) | 3<-(I0-L3,I1-L4,I2-L5)], |nor_conv_1x1~0|+|avg_pool_3x3~0|none~1|+|skip_connect~0|nor_conv_1x1~1|nor_conv_3x3~2|
      (layers): ModuleList(
        (0): ReLUConvBN(
          (op): Sequential(
            (0): ReLU()
            (1): Conv2d(16, 16, kernel_size=(1, 1), stride=(1, 1), bias=False)
            (2): BatchNorm2d(16, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
          )
        )

The issue is, would it cause unreliable learning when a custom dataset has small objects in it?
The input size of my train_loader and test loader for a custom datasets are having bigger size like `torch.Size([32, 3, 64, 64])` while getting passed to the networks with Conv2d(3, **_16_**, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)

+ Do you consider changing config['C'] = 32 or any other values is effective way for increasing accuracy for a custom datasets with resolution like 128x128 or 256x256?

**Comment by VascoLopes** [comment]
Dear @Mshz2,

What you reefer as "input size of 16", is actually the output size (channels) of the Conv2D layers. In PyTorch, the first parameter of a Conv2D defines the input channels, and the second one, the output channels. You can find more information about this on the PyTorch documentation.

Regarding the question about changing the parameters of the architecture to accommodate different input sizes. You can certainly change the models to accommodate any needs, but in NAS-Bench-201 you will only get the performance of the exact architectures given. More, the output channels of a Conv2D might influence the final performance, but for your specific question, layers that reduce the size of the input would be preferable, to accommodate that differences.

Hope this helps you
